Day-II-Calculator/calculator/src/components/ButtonComponents/ActionButton.js

```diff
@@ -1,2 +1,13 @@
 import React from 'react';
-import './Button.css';
+
+const ActionButton = props => {
+    return (
+        <>
+            <button className="action-button" value={props.text}>
+                {props.text}
+            </button>
+        </>
```

Comment:
Why use a fragment here? `button` doesn't need a parent, and fragments have no styling effect since they're not actually DOM elements.